sklearn/metrics/classification.py

```diff
@@ -1393,7 +1393,7 @@ class 2       1.00      0.67      0.80         3
         labels = np.asarray(labels)
 
     if target_names is not None and len(labels) != len(target_names):
-        warnings.warn("labels size, {}, does not match size of target_names, {}".format(len(labels), len(target_names))
```

Comment:
The line is to long now. You can also run flake8 or pyflakes locally to find these errors without running the continuous integration. I would recommend installing a flake8 plugin for your editor / IDE.